recipes/default.rb

```diff
@@ -8,3 +8,4 @@
 include_recipe 'curl'
 
 elasticsearch_user 'elasticsearch'
+elasticsearch_install 'elasticsearch'
```

Comment:
I'm not very familiar with the LWRP based DSL (shame on me I know :), can we document somewhere how to use the `elasticsearch_install` method? Ie. why do we need to pass `elasticsearch` as a parameter here? Can the user provide eg. a specific `version:` here?